tests/integration/test_schema_cache.py

```diff
@@ -0,0 +1,43 @@
+import os
+
+from app import cache, settings
+from tests.integration.create_token import create_token
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestApplicationVariables(IntegrationTestCase):
+
+    def setUp(self):
+        settings.EQ_DEV_MODE = False
+
+        for key_name, dev_location in settings._KEYS.items():
+            path = os.getenv(key_name, dev_location)
+            vars(settings)[key_name] = settings.get_key(path)  # assigns attribute to this module
+
+        for password_name, dev_default in settings._PASSWORDS.items():
+            password = os.getenv(password_name, dev_default)
+            vars(settings)[password_name] = password  # assigns attribute to this module
+
+        IntegrationTestCase.setUp(self)
+
+    def tearDown(self):
+        settings.EQ_DEV_MODE = True
+        IntegrationTestCase.tearDown(self)
+
+    def test_schema_is_cached(self):
+        with self.application.app_context():
+
+            self.assertEqual(len(cache.cache._cache), 0)
+
+            token = create_token('star_wars', '0')
+            resp = self.client.get('/session?token=' + token.decode(), follow_redirects=True)
+
+            self.assertEquals(resp.status_code, 200)
+
+            self.assertNotEquals(cache.cache._cache['app.utilities.schema.load_and_parse_schema_memver'], None)
```

Comment:
`cache.cache._cache['app.utilities.schema.load_and_parse_schema_memver']` could be changed to use the public method cache.get('app.utilities.schema.load_and_parse_schema_memver')